ivadomed/config/example_loader_v2_configs.py

```diff
@@ -0,0 +1,168 @@
+                        [["sub-03/ses-02/anat/sub-03_ses-02_flip-1_mt-off_MTS.nii",
+                          "sub-03/ses-02/anat/sub-03_ses-02_flip-1_mt-on_MTS.nii"],
+                         ["derivatives/labels/sub-03/ses-02/anat/sub-03_ses-02_mt-off_MTS_lesion-manual-rater1.nii"]],
+                        [["sub-03/ses-03/anat/sub-03_ses-03_flip-1_mt-off_MTS.nii",
+                          "sub-03/ses-03/anat/sub-03_ses-03_flip-1_mt-on_MTS.nii",
+                          "sub-03/ses-03/anat/sub-03_ses-03_flip-1_mt-on_MTS.nii"],
+                         ["derivatives/labels/sub-03/ses-03/anat/sub-03_ses-03_mt-off_MTS_lesion-manual-rater1.nii"]],
+                        [["sub-03/ses-04/anat/sub-03_ses-04_flip-1_mt-off_MTS.nii", ],
+                         ["derivatives/labels/sub-03/ses-04/anat/sub-03_ses-04_mt-off_MTS_lesion-manual-rater1.nii"]],
+                    ],
+                    DataloaderKW.MISSING_FILES_HANDLE: FileMissingHandleKW.SKIP,
+                    DataloaderKW.EXCESSIVE_FILES_HANDLE: FileExcessiveHandleKW.USE_FIRST_AND_WARN,
+            },
+            DataloaderKW.PATH_DATA: path_mock_data,
+            DataloaderKW.EXPECTED_INPUT: 2,
+            DataloaderKW.EXPECTED_GT: 1,
+
+}
+
+example_2i1o_all_dataset_groups_config_json: dict = {
+    DataloaderKW.DATASET_GROUPS: [
+        example_file_dataset_group_config_json,
+        {
+            DataloaderKW.DATASET_GROUP_LABEL: "FileDataSetGroup2",
+            DataloaderKW.TYPE: "FILES",
+            DataloaderKW.TRAINING: {
+                    DataloaderKW.IMAGE_GROUND_TRUTH: [
+                        [["sub-04/ses-01/anat/sub-04_ses-01_flip-1_mt-off_MTS.nii",
+                          "sub-04/ses-01/anat/sub-04_ses-01_flip-1_mt-on_MTS.nii"],
+                         ["derivatives/labels/sub-04/ses-01/anat/sub-04_ses-01_mt-off_MTS_lesion-manual-rater1.nii"]],
+                        [["sub-04/ses-02/anat/sub-04_ses-02_flip-1_mt-off_MTS.nii",
+                          "sub-04/ses-02/anat/sub-04_ses-02_flip-1_mt-on_MTS.nii"],
+                         ["derivatives/labels/sub-04/ses-02/anat/sub-04_ses-02_mt-off_MTS_lesion-manual-rater1.nii"]],
+                        [["sub-04/ses-03/anat/sub-04_ses-03_flip-1_mt-off_MTS.nii",
+                          "sub-04/ses-03/anat/sub-04_ses-03_flip-1_mt-on_MTS.nii",
+                          "sub-04/ses-03/anat/sub-04_ses-03_flip-1_mt-on_MTS.nii"],
+                         ["derivatives/labels/sub-04/ses-03/anat/sub-04_ses-03_mt-off_MTS_lesion-manual-rater1.nii"]],
+                        [["sub-04/ses-04/anat/sub-04_ses-04_flip-1_mt-off_MTS.nii", ],
+                         ["derivatives/labels/sub-04/ses-04/anat/sub-04_ses-04_mt-off_MTS_lesion-manual-rater1.nii"]],
+                    ],
+                    DataloaderKW.MISSING_FILES_HANDLE: FileMissingHandleKW.SKIP,
+                    DataloaderKW.EXCESSIVE_FILES_HANDLE: FileExcessiveHandleKW.USE_FIRST_AND_WARN,
```

Comment:
This is a detail but I could not find where this warning is raised/did not see it in my test (it may not be implemented yet, which if fine as well, just wondering).

Reply:
Good point. just implemented it now. (had the warning for LESS but missed MORE for some reason)